src/executables/master_heartbeat.h

```diff
@@ -118,6 +118,27 @@ enum HB_PROC_STATE
 
 #define HB_REPLICA_PRIORITY                     0x7FFF
 
+/* Heartbeat resource process detail state.
+ * These states are to provide detailed information about server process
+ * from start (HB_PSTATE_STARTED) to the provision of the service (HB_PSTATE_REGISTERED_*).
+ * When the server is ready to provide a service, then it does not need to show a detail state.
+ */
+enum HB_SERVER_STATE
+{
+  HB_SERVER_STATE_INIT = 0,
+  HB_SERVER_STATE_RECOVERY = 1,
+  HB_SERVER_STATE_RECOVERED = 2,	/* only for PS */
+  HB_SERVER_STATE_CATCH_UP = 3,	/* only for PS */
+  HB_SERVER_STATE_READY = 4	/* When the server is ready, then no detail state to be shown to user */
+};
+
+#define HB_SERVER_STATE_INIT_STR                "(initial)"	/* ex) state started(initial) */
```

Comment:
```suggestion
#define HB_SERVER_STATE_INIT_STR                "initial"	/* ex) state started(initial) */
```
I think the parenthesis is a part of "format". How about making these with state lie "initial"?

Reply:
Agreed. I added the parentheses to simplify the code that print the state value. I will modify it.